include/zephyr/drivers/flash.h

```diff
@@ -58,9 +58,8 @@ struct flash_pages_layout {
  */
 struct flash_parameters {
 	const size_t write_block_size;
-	/* Device capabilities. Only drivers are supposed to set the
-	 * capabilities directly, users need to call the FLASH_CAPS_
-	 * macros on pointer to flash_parameters to get capabilities.
+	/* User code should call flash_params_get_ functions on flash_parameters
+	 * to get capabilities, rather than accessing object contents directly.
```

Comment:
it would probably make sense to make sure all the members of this struct actually have proper doxygen comments? `/** ...` ?
Also, not familiar with the Flash API myself but since the comment above mentions "Contents of this structure suppose to be filled in during flash device init ..." should the members in fact be "INTERNAL_HIDDEN" as they're maybe not meant to be API anyway?

Reply:
OK, will fix that.